app/views/layouts/application.html.erb

```diff
@@ -8,10 +8,14 @@
     <meta name="tiny-mce-assets-url" content="<%= tiny_mce_assets_path %>">
     <meta name="highlightjs-url" content="<%= asset_path('highlightjs-github-theme.css') %>">
     <%= stylesheet_link_tag 'application', media: 'all' %>
-    <%= javascript_include_tag 'application' %>
-    <%= javascript_pack_tag 'application' %>
 
+    <%= javascript_include_tag 'jquery_bundle' %>
+
+    <%= javascript_pack_tag 'tiny_mce' %>
+    <%= stylesheet_pack_tag 'tiny_mce' %>
```

Comment:
Maybe move to HTML partial and include this partial only on required pages? And in add-ons just override partially. We need tinymce only at 3 pages.